dependencies.yaml

```diff
@@ -573,13 +595,6 @@ dependencies:
       - output_types: conda
         packages:
           - libcuspatial-tests==24.10.*,>=0.0.0a0
```

Comment:
I’m not sure if I like this structure where we have a dependency list that defines libcuspatial-tests. In libcudf tests we list the libcudf-tests package explicitly rather than using a dependency list. https://github.com/rapidsai/cudf/blob/7018a33be752da9363db5431560d8d12bf378920/ci/test_cpp_common.sh#L34

The *-tests conda packages are kind of special in that they are only CI artifacts and thus never available from any “proper” conda channels, only local channels of CI built artifacts. For that reason I am hesitant to use RDFG to define them

Reply:
Hmmmm. The alternate proposal I had in mind (removing the test_libcuspatial dependency list and installing libcuspatial-tests manually) would not make sense for this repo, since we’ve moved to the “single step solve” for C++ tests in cuspatial.  https://github.com/rapidsai/cuspatial/blob/0e9d36cff2c8e4fcfdd9e88e7ec3fa07a6db6184/ci/test_cpp.sh#L17

Maybe we do have the right approach and no changes are needed…

Reply:
yep exactly, the use of that `--prepend-channel` pattern here is why I left this as-is.